apps/website/src/components/show-and-tell/ShowAndTellEntry.tsx

```diff
@@ -35,6 +43,18 @@ type ShowAndTellEntryProps = {
   showPermalink?: boolean;
 };
 
+const parseOptions: HTMLReactParserOptions = {
+  replace: (node) => {
```

Comment:
(As a note for the future, this lets us far more easily implement any sanitization if we need it)